presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -88,12 +97,23 @@ Property Name                                        Description
 
 ==================================================== ============================================================
 
+Hadoop catalog
+^^^^^^^^^^^^^^
+
+Iceberg connector supports Hadoop as Catalog
+
 .. code-block:: none
 
     connector.name=iceberg
-    iceberg.catalog.type=nessie
-    iceberg.catalog.warehouse=/tmp
-    iceberg.nessie.uri=https://localhost:19120/api/v1
+    hive.metastore.uri=hostname:port
+    iceberg.catalog.type=hadoop
+
+.. note::
+
+    To use Hadoop catalog with Iceberg table, it requires the ``hive.metastore.uri`` property, even
+    though it doesn't rely on metastore for metadata when using Hadoop Catalog. You can provide
+    a placeholder with ``hive.metastore.uri=thrift://localhost:9083``. This will be fixed in a
+    future release of Presto (:issue:`20579`).
```

Comment:
@agrawalreetika Can you please double check if this issue display is correct? It shows up as this picture in my browser
<img width="619" alt="Screenshot 2023-08-16 at 6 15 22 PM" src="https://github.com/prestodb/presto/assets/33299678/0d667dd3-b6d1-4e25-9b55-af7d775fb33d">

Reply:
Verified, after compiling on web UI it's showing correct issue link - 
![image](https://github.com/prestodb/presto/assets/7887476/a0744632-c7ce-4065-8cba-37859899f48a)